lib/devise/jwt/cookie.rb

```diff
@@ -14,9 +14,9 @@ module JWT
     module Cookie
       extend Dry::Configurable
 
-      setting :name, 'access_token'
-      setting :secure, true
-      setting :domain
+      setting :name, default: 'access_token'
+      setting :secure, default: true
+      setting :domain, default: 'localhost:3000'
```

Comment:
Will this work on prod like this?
Error is present there as well https://app.datadoghq.eu/logs?query=%22module%3AJWT%22&cols=&index=&messageDisplay=inline&stream_sort=time%2Cdesc&viz=stream

Reply:
> But if we actually set the `domain` then it won't need a default.

This actually makes sense. The other 2 have defaults but not the `default` word so they should fix the errors we are getting. Where as domain doesn't need it and can be set in the devise initialiser file.